README.md

```diff
@@ -77,82 +77,13 @@ Manuals for the most common use cases of updating website content are available
 
 ### Testing
 
-[Cypress](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell) is used to run End-To-End tests. Tests are located in the `cypress/e2e` folder and can be run with:
+Most tests run automatically when a pull request is submitted. If you are making changing which affect hyperlinks in the web it is recommended to run the links test locally before submitting a pull request or making changes to it.
```

Comment:
```suggestion
Most tests run automatically when a pull request is submitted. If you are making changes which affect hyperlinks in the web it is recommended to run the links test locally before submitting a pull request or making changes to it.
```

If you are making changing -> If you are making changes

Reply:
Thanks for noticing the typo!